QUICKSTART.md

```diff
@@ -334,6 +340,7 @@ which should yield:
   </body>
 </html>
 ```
+NOTE: if you receive the error: 'curl: (52) Empty reply from server', wait 2 minutes and attempt curl again.  It takes a minute for the Apache HTTP Server to synchronize and start.
```

Comment:
I wonder if "2 minutes" is too specific.  We can not promise that 2 minutes is enough.  If we promise one specific obviously large enough time, that will be wasteful and embarrasing in almost all cases. Things seem to run more slowly for me than for @francostellari .  I can attest that I had to wait a while, but did not measure how long.  Maybe say something more vague, like keep retrying, it should work in a small number of minutes?

Reply:
see line 361 and 362 below